src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -28,6 +28,10 @@
 #define USBD_PRODUCT_STRING     "CrazyBee F3 FR"
 #endif
 
+// clear up some flash space
+#undef USE_PWM
```

Comment:
- the cleanup should be done in a separate pull request;
- the list used for cleanup should be the same in all targets, or else managing flash space on F3 will be even more painful as it already is;
- #6444 already takes care of it (but happy to look into using the reductions suggested by you instead, if they are sufficient).

Reply:
Noted. I reverted the last commit and will submit another PR with the same set of (ideally rarely used) features removed

Reply:
Hold on, repurposing of #6444 already in progress.